docs/UserGuide.md

```diff
@@ -464,7 +515,14 @@ Format: `fee n/STUDENT_NAME m/MONTH y/YEAR`
 * `YEAR` should be a positive integer between 1970 and 2037 inclusive
 
 Example:
-* `fee n/John Doe m/1 y/2021` returns John Doe monthly fee for January 2021
+1. You want to get the monthly fee payable by `Alex Yeoh` for March 2021.
```

Comment:
Maybe we shouldn't use the code snippet format cause its not a command or response, perhaps bolding would work?

Reply:
Ah right! Agreed! Have updated in the latest commit.